geoportailv3/static/js/catalog/catalogdirective.js

```diff
@@ -114,9 +114,7 @@ app.CatalogController.prototype.setTree_ = function() {
        * @param {Object} tree Tree object for the theme.
        */
       function(tree) {
-        if (!goog.isNull(tree)) {
-          this['tree'] = tree;
-        }
```

Comment:
Are you sure about removing this? I think there was a good reason for this test.